easybuild/easyconfigs/s/SLEPc/SLEPc-3.7.4-CrayIntel-17.08-complex-64.eb

```diff
@@ -0,0 +1,38 @@
+# @author: gppezzi (CSCS)
+easyblock = 'ConfigureMake'
+
+name = 'SLEPc'
+version = "3.7.4"
+versionsuffix = '-complex-64'
+
+homepage = 'http://slepc.upv.es/'
+description = """SLEPc (Scalable Library for Eigenvalue Problem Computations) is a software library for the solution of
+ large scale sparse eigenvalue problems on parallel computers. It is an extension of PETSc and can be used for either
+ standard or generalized eigenproblems, with real or complex arithmetic. It can also be used for computing a partial
+ SVD of a large, sparse, rectangular matrix, and to solve quadratic eigenvalue problems."""
+
+toolchain = {'name': 'CrayIntel', 'version': '17.08'}
+#toolchainopts = {'usempi': True}
+
+source_urls = ['http://slepc.upv.es/download/distrib/']
+sources = [SOURCELOWER_TAR_GZ]
+
+dependencies = [
+      ('cray-petsc-complex-64/3.7.6.2', EXTERNAL_MODULE),
```

Comment:
Please update metadata file accordingly with a new entry for this version.

Reply:
Data has been added